spec.html

```diff
@@ -24641,18 +24641,18 @@ <h1>Object ( . . . )</h1>
       <p>See <emu-xref href="#sec-object-constructor"></emu-xref>.</p>
     </emu-clause>
 
-    <!-- es6num="18.3.17" -->
-    <emu-clause id="sec-constructor-properties-of-the-global-object-proxy">
-      <h1>Proxy ( . . . )</h1>
-      <p>See <emu-xref href="#sec-proxy-constructor"></emu-xref>.</p>
-    </emu-clause>
-
     <!-- es6num="18.3.18" -->
     <emu-clause id="sec-constructor-properties-of-the-global-object-promise">
       <h1>Promise ( . . . )</h1>
       <p>See <emu-xref href="#sec-promise-constructor"></emu-xref>.</p>
     </emu-clause>
 
+    <!-- es6num="18.3.17" -->
+    <emu-clause id="sec-constructor-properties-of-the-global-object-proxy">
+      <h1>Proxy ( . . . )</h1>
+      <p>See <emu-xref href="#sec-proxy-constructor"></emu-xref>.</p>
+    </emu-clause>
+
```

Comment:
Is it intentional that 18.3.17 now goes after 18.3.18?

Reply:
It's intentional to put 'Promise' before 'Proxy' -- that's alphabetical order.

The es6nums just come along for the ride. The fact that they're now 'out of order' is (a) unimportant, and (b) because ES6 inserted 'Proxy' before 'Promise'.